pubspec.lock

```diff
@@ -163,6 +163,13 @@ packages:
     description: flutter
     source: sdk
     version: "0.0.0"
+  get_it:
```

Comment:
Can you please share a link for this?
And also, what are the reasons you've decided to use a dependancy for singleton implementation?

Reply:
https://pub.dev/packages/get_it

im read an article about DI in Flutter and saw get_it package. Read on pub.dev about get_it and I understood that this is what I need.